src/app/pages/Sandbox/Editor/Content/Header/ShareView.js

```diff
@@ -296,11 +343,25 @@ class ShareView extends React.PureComponent {
                       setValue={this.setHideNavigation}
                     />
                     <PaddedPreference
+                      title="Single file entry script"
```

Comment:
that also means 3 buttons for "Default View" could go too. ..i dont know how i feel about it. im sure ur more attached. i do find them useful. but only because I know what they mean. Truthfully, those 3 buttons i couldnt guess what they did until i clicked them eventually. And still it was confusing. I know it's supposed to be obvious, but for some reason it's not. They kinda just sit and look pretty.

It's key to ur brand though, so i wouldnt remove those buttons. I just think getting this embed right will be important. 1, it's less to manage. 2, if it's easy, it means more sharing, which means more traffic.

So here's what i would do: keep the 3 switches on the full editor, as it's good for the brand, and it is useful, and i was just dumb and probably in the minority for it not making sense at first.

However, for the embed, dont offer the 3 default view switches. Instead just always default to split screen. Period. That's your brand there--u want people to see CODE + AN APP when they see ur embed. In addition, if ur not in split mode, the screen is too wide for the alloted space. It only makes sense in split mode. Nobody is making super wide screen apps for example--it looks bad. Split screen looks great. Default to split screen, remove those buttons.

I'd also say: forget the HIDE NAVIGATION BAR. U want that--even if many sandboxes just use `/`. The reason is it's a visual que that the user is looking at a demo web browser. Plus, with RFR, I promise people will start using the URL more. It makes it stupid simple to URL-ize any redux app. ..But also u should promote using URLs since it's good for everyone. SPAs are dead. They are no good. The URL makes it nice to see demos in a few contexts and let users go back and forth between them. So it's good to have ur sandbox creators thinking about creating different URLs for their app, rather than having it solely be interactive.

Anyway, that's the case for always keeping the address bar. The case against it is 48 pixels in height lost. One more case *for* it: the address bar makes the preview vertically even with the code editor. 

Forget the font-size too.

All this stuff is friction. Here's the concept: YOU ALREADY HAD A USER WILLING TO SHARE! NOW THEY SEE THIS, THEY ARE OVERLOADED WITH DECISIONS AND A LARGE PERCENTAGE NOW DO NOT SHARE.

The only thing i dont know about is the "auto resize" feature for medium. What is that?

Lastly, here's what I would do: 
![screen shot 2017-07-12 at 9 19 20 am](https://user-images.githubusercontent.com/154732/28128041-360390e4-66e3-11e7-8e19-f3c8267e8b5d.png)
That's a shitty example, but the idea is find somewhere else to move that shit, remove the ShareView modal, and just put the damn URL to share there. It already automatically embeds on Medium. 

For the embed/iframe URL, instead just write code that detects ur being run in an iframe. So now u just got rid of 2 of the 3 URL fields and the iframe textarea (though i understand u want ur border radius, and ideally we have those iframe sandbox values...But anyway, ur left with markdown and html for the button and maybe the iframe. Put that as a secondary share option for the hyper-motivated--***but the overall idea is make the main URL just one URL and stupid simple and ever-present to copy paste.*** That's why im suggesting to put it in the nav bar. But what's better is: a footer that contains just this.